cmd/tendermint/commands/compact.go

```diff
@@ -0,0 +1,70 @@
+package commands
+
+import (
+	"path/filepath"
+	"sync"
+
+	"github.com/spf13/cobra"
+	"github.com/syndtr/goleveldb/leveldb"
+	"github.com/syndtr/goleveldb/leveldb/opt"
+	"github.com/syndtr/goleveldb/leveldb/util"
+
+	"github.com/tendermint/tendermint/config"
+	"github.com/tendermint/tendermint/libs/log"
+)
+
+func MakeCompactDBCommand(cfg *config.Config, logger log.Logger) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "experimental-compact-db",
```

Comment:
```suggestion
		Use:   "experimental-compact-leveldb",
```

Reply:
Do you think we need to specify goleveldb (instead of cleveldb)